data/food/ecosystemic_services/feed.json

```diff
@@ -1,151 +1,151 @@
+    "c88d387e-8435-4741-b742-0094dbdcee45": 0.175,
+    "dc872330-8d35-4b02-bf45-4481b89a093e": 0.349
   },
-  "2bf307e8-8cb0-400b-a4f1-cf615d9e96f4": {
-    "305cc005-eb5c-4e94-b5cf-9aa21c619577": 0.071,
-    "a98b8776-a96d-48e9-b218-976f5452907a": 0.0477,
-    "585c8bb1-1aaa-4c33-a5c1-84c886bb7ec3": 0.0384,
-    "ba97e4b9-1fac-4f92-bed1-9889e3ae631d": 0.579,
-    "c88d387e-8435-4741-b742-0094dbdcee45": 0.814
+  "8fb5024c-ceec-4201-b069-ea253c537ad6": {
+    "086b58da-1c4c-40a8-9327-37477d04cfba": 0.87,
+    "38788025-a65e-4edf-a92f-aab0b89b0d61": 1.23,
+    "ab087863-f311-4066-8e93-4d7f6e0db2b2": 0.87,
+    "be9a2aa4-4c8b-4801-abcf-01ba1581f053": 1.9
   },
-  "d79e0fce-8633-4189-a937-7b8010abafed": {
-    "38788025-a65e-4edf-a92f-aab0b89b0d61": 2.94,
-    "141f76d7-fe36-4f17-b7da-043ac884a309": 0.91,
-    "ba97e4b9-1fac-4f92-bed1-9889e3ae631d": 12,
-    "c88d387e-8435-4741-b742-0094dbdcee45": 25
+  "9cbc31e9-80a4-4b87-ac4b-ddc051c47f69": {
+    "141f76d7-fe36-4f17-b7da-043ac884a309": 0.35,
+    "38788025-a65e-4edf-a92f-aab0b89b0d61": 0.567,
+    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 0.977
   },
-  "e07f7703-c4f7-479e-a8ac-bb73d3f08c78": {
-    "a98b8776-a96d-48e9-b218-976f5452907a": 2.94,
-    "3c16dc23-021d-45ef-8805-c77b043b041a": 1.5,
-    "ba97e4b9-1fac-4f92-bed1-9889e3ae631d": 14,
-    "c88d387e-8435-4741-b742-0094dbdcee45": 67
+  "9db6b0d1-941a-4dda-a347-b7a7fee8fd46": {
+    "141f76d7-fe36-4f17-b7da-043ac884a309": 0.35,
+    "38788025-a65e-4edf-a92f-aab0b89b0d61": 0.83,
+    "ba97e4b9-1fac-4f92-bed1-9889e3ae631d": 3.3,
+    "c88d387e-8435-4741-b742-0094dbdcee45": 24.51,
+    "dc872330-8d35-4b02-bf45-4481b89a093e": 2.23
   },
   "a360871c-fc35-4cb3-af2c-5eeb8bd6f984": {
-    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 1.9,
-    "38788025-a65e-4edf-a92f-aab0b89b0d61": 1.23,
     "141f76d7-fe36-4f17-b7da-043ac884a309": 0.87,
-    "ab087863-f311-4066-8e93-4d7f6e0db2b2": 0.87
-  },
-  "f94df72e-20fd-4cc5-ab64-ce3921649d58": {
-    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 1.9,
     "38788025-a65e-4edf-a92f-aab0b89b0d61": 1.23,
-    "086b58da-1c4c-40a8-9327-37477d04cfba": 0.87,
+    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 1.9,
     "ab087863-f311-4066-8e93-4d7f6e0db2b2": 0.87
   },
   "b19b0dc0-3478-46a0-b8a1-3225ea939ff1": {
     "741c0970-2823-430a-b4e4-e61ce08ea95a": 2.91,
-    "a98b8776-a96d-48e9-b218-976f5452907a": 0.63,
     "92d34545-6952-466e-bf6d-c1f1f6069b48": 0.87,
+    "a98b8776-a96d-48e9-b218-976f5452907a": 0.63,
     "ab087863-f311-4066-8e93-4d7f6e0db2b2": 0.87
   },
-  "8fb5024c-ceec-4201-b069-ea253c537ad6": {
-    "be9a2aa4-4c8b-4801-abcf-01ba1581f053": 1.9,
-    "38788025-a65e-4edf-a92f-aab0b89b0d61": 1.23,
-    "086b58da-1c4c-40a8-9327-37477d04cfba": 0.87,
-    "ab087863-f311-4066-8e93-4d7f6e0db2b2": 0.87
+  "b2e75be9-65b8-487d-b5b3-f9c2aa047cf7": {
+    "141f76d7-fe36-4f17-b7da-043ac884a309": 0.453,
+    "38788025-a65e-4edf-a92f-aab0b89b0d61": 0.644,
+    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 1.14
```

Comment:
Please note that this makes very hard to ensure that no values have been altered after keys have been reordered